### ISSUE TITLE
dx: I want to see what the exported API is for other modules

### ISSUE THREAD

**Issue by matt2e** [block/ftl#2394]
Use case:
- I am building a module and need to interface with other modules
- When i run `ftl schema get` I get a long list of things, many of them internal to each module (secrets, config, dbs, unexported data types and verbs). This makes it hard to focus on the interface I am trying to glean for each module

Maybe something like ` ftl schema get --exported`? Or would developers not be looking at the cli for this?

**Comment by alecthomas** [comment]
I wonder if we should default to only exported symbols?

**Comment by wesbillman** [comment]
Yeah I like the idea of defaulting to only exported as well and maybe have an `--all` or `--verbose` flag for getting everything.
